__tests__/fdir.test.js

```diff
@@ -130,22 +135,29 @@ describe.each(["withPromise", "sync"])("fdir %s", (type) => {
   });
 
   test("getting files from restricted directory shouldn't throw (suppressErrors)", async () => {
-    const api = new fdir().crawl("/etc");
+    const secureDir = 
+      process.platform === 'win32' 
+        ? 'C:\\Windows'
```

Comment:
Are we sure it will always be `C:\\Windows`?

Reply:
I would say it's safe to assume Windows is installed on the `C:` drive. While it is technically possible to run windows off of another drive, this is extremely rare. This could be addressed in the future once a contributor actually needs the unit tests to support crawling the `Windows` directory from another drive.